### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by scott-confluent**
This is a first round fix for the bugs we've found. This is for all high priority bugs unless otherwise noted in the google sheet we're tracking them with

### home page
-  spelling
- re-introduce see full list tooltip

### docs
- remove version banner
- fixed Markup issues on https://kafka.apache.org/documentation/streams/developer-guide/dsl-api.html
- removed duplicate headline and initial section on streams running app page

### powered by
- fixed JSON issue in page

### CODE REVIEW

File: 25/streams/developer-guide/write-streams.html [hunk 1]
```diff
@@ -90,22 +90,22 @@
...
-    <artifactId>kafka-streams</artifactId>
-    <version>{{fullDotVersion}}</version>
-</dependency>
-<dependency>
-    <groupId>org.apache.kafka</groupId>
-    <artifactId>kafka-clients</artifactId>
-    <version>{{fullDotVersion}}</version>
-</dependency>
+          <pre class="line-numbers"><code class="language-xml">&lt;dependency>
+    &lt;groupId>org.apache.kafka&lt;/groupId>
```
Comment (guozhangwang) [inline 25/streams/developer-guide/write-streams.html:94]: Should the `>` be replaced too?

File: 25/documentation.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
     <!--#include virtual="toc.html" -->
   </div>
 	<div class="right">
-    <!--#include virtual="../includes/_docs_banner.htm" -->
```
Comment (guozhangwang) [inline 25/documentation.html:34]: Ditto for other 25/xyx.html pages: does the same issue exist in older versions (24, 23...)?

**Review by guozhangwang — COMMENTED** [review]
made a pass , cc @ijuma @bbejeck for another pass.

File: 25/streams/developer-guide/write-streams.html (hunk 1 above)
Comment (scott-confluent) [reply 25/streams/developer-guide/write-streams.html:94]: theoretically it should be generated with html entities for both gt and lt, but the opening is the only one that matters for highlighting the syntax. it's not an easy thing to find/replace so we'd have to do an audit of all the docs.

File: 25/documentation.html (hunk 2 above)
Comment (scott-confluent) [reply 25/documentation.html:34]: I believe the older versions SHOULD have this banner.

**Pull request merged**